draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -200,11 +201,10 @@ where `<unique-N>` is a label that tags each record in the collection.
 Member node labels carry no informational meaning beyond labeling member zones.
 A changed label may indicate that the state for a zone needs to be reset (see (#zonereset)).
 
-Having the zones uniquely tagged with the `<unique-N>` label ensures that
-additional RRs can be added at or below the member node, for signifying
-member-zone-specific information (described below). Further, if member zones
-do not share a PTR RRset, the list of member zones can be split over multiple
-DNS messages in a zone transfer.
+Having the zones uniquely tagged with the `<unique-N>` label ensures that additional RRs can be added below the member node (see (#properties)).
+Further, if member zones do not share a PTR RRset, the list of member zones can be split over multiple DNS messages in a zone transfer.
+
+A catalog zone consumer MUST not process and ignore PTR RRsets with more than a single record.
```

Comment:
it's unclear whether `not` refers to `ignore`. Better: `MUST ignore` (no `process`)

Reply:
agree. Done in commit 05736f23